apps/meteor/client/views/marketplace/helpers.ts

```diff
@@ -7,7 +7,7 @@ import { Utilities } from '../../../app/apps/lib/misc/Utilities';
 import { t } from '../../../app/utils/client';
 import { dispatchToastMessage } from '../../lib/toast';
 
-export const appEnabledStatuses = [AppStatus.AUTO_ENABLED, AppStatus.MANUALLY_ENABLED, AppStatus.INITIALIZED];
```

Comment:
Removing this will probably bring back this bug: https://github.com/RocketChat/Rocket.Chat/pull/27330

Reply:
Good catch. However now we cannot consider `INITIALIZED` as "Enabled" anymore. Do you have a suggestion on how to move forward with this?

Reply:
We could create an "initialized" status for the UI to better represent the actual apps engine status. Wdyt?

Reply:
We could create an "initialized" status for the UI to better represent the actual apps engine status. Wdyt?

Reply:
Makes sense to me